src/libs/Altinn.Authorization.Api.Contracts/src/Altinn.Authorization.Api.Contracts/Altinn.Authorization.Api.Contracts.csproj

```diff
@@ -12,4 +12,8 @@
     <PackageReference Include="Altinn.Urn.Swashbuckle" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\apps\Altinn.AccessManagement\src\Altinn.AccessMgmt.PersistenceEF\Altinn.AccessMgmt.PersistenceEF.csproj" />
```

Comment:
Denne må fjernes. Er nok pga bruk av EF modeller inne i noen av Dto'ene. Altså må det lages Dto'er og mapping for de modellene som mangler